core/src/main/scala/org/bitcoins/core/protocol/Address.scala

```diff
@@ -190,7 +190,7 @@ object Bech32Address extends AddressFactory[Bech32Address] {
       np: NetworkParameters): Try[Bech32Address] =
     spk match {
       case witSPK: WitnessScriptPubKey =>
-        Bech32Address.fromScriptPubKey(witSPK, np)
+        Success(Bech32Address(witSPK, np))
```

Comment:
This function was an infinite loop so I fixed it

Reply:
yikes!